src/main/scala/com/decodified/scalassh/SshClient.scala

```diff
@@ -53,9 +52,10 @@ final class SshClient(val config: HostConfig) extends ScpTransferable {
         execWithSession(command, session)
       }
     }
+  def logCmd(command: Command): String = if (command.safe) command.command else "Sensitive data, not logged"
```

Comment:
This appears to me as a bit coarse-grained.

What we want to achieve is not to log sensitive data, which are usually _arguments_ to commands rather than the commands themselves.

Reply:
also gone, at first i thought i would need it in more than on place, but not the case.